modules/drivers/presto/src/metabase/driver/presto.clj

```diff
@@ -119,8 +124,9 @@
   [details query]
   {:pre [(map? details)]}
   (ssh/with-ssh-tunnel [details-with-tunnel details]
-    (let [{{:keys [columns data nextUri error id]} :body} (http/post (details->uri details-with-tunnel "/v1/statement")
-                                                                     (assoc (details->request details-with-tunnel) :body query, :as :json))]
+    (let [{{:keys [columns data nextUri error id infoUri]} :body} (http/post (details->uri details-with-tunnel "/v1/statement")
+                                                                     (assoc (details->request details-with-tunnel)
```

Comment:
https://github.com/bbatsov/clojure-style-guide#vertically-align-fn-args

Reply:
![image](https://user-images.githubusercontent.com/6198952/58117747-a51e7d00-7bd5-11e9-840b-0fd2da9e82ab.png)

Not sure the best approach here since it runs over the line length in the rest of the file.

Reply:
In that case you should put the args on new lines and indent them all by one space. See https://github.com/bbatsov/clojure-style-guide#one-space-indent

Alternatively you can put the right hand side of the let binding on a new line like this

```clj
(let [{{:keys [columns data nextUri error id infoUri]} :body}
      (http/post ...)]
  ...)

https://github.com/bbatsov/clojure-style-guide#no-blank-lines-within-def-forms